src/sdk/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "odp-sdk"
-version = "0.4.10"
+version = "0.7.0"
```

Comment:
The `Makefile` sets this value during the publish-pipeline. Does it also have to be explicitly set like this?

Reply:
From what I understood reading the Makefile, it only updates the main repository, not the subdirectories. I guess that as odp-sdk existed beforehand, it still has some impact.

Reply:
The makefile updates all the pyproject.toml-files, however the changes will not be committed. Do we need the changes to be committed?

Reply:
It seems like that. Otherwise, when you do a poetry install the version gets downgraded

Reply:
This is the origin of the problem: https://github.com/C4IROcean/odp-sdk-python/pull/83/files#diff-f53a023eedfa3fbf2925ec7dc76eecdc954ea94b7e47065393dbad519613dc89R498 In the poetry.lock, the version of the odp-sdk was set to an incorrect number

Reply:
Maybe we can set the value `1.0.0a0.dev0` in the repo? This should indicate a `1.0.0` dev pre-release. (More info here: https://peps.python.org/pep-0440/#developmental-releases)

We could also use a placeholder-value such as `99.0.0.dev0`

Reply:
Where this should be set? In poetry or in the repo itself?

Reply:
All `pyproject.toml`-files